vaadin-iron-list-flow-parent/vaadin-iron-list-flow-integration-tests/src/main/java/com/vaadin/flow/component/ironlist/tests/IronListView.java

```diff
@@ -202,17 +190,14 @@ private void createChuckNorrisFacts() {
                         + "border:1px solid lightgray; border-radius:5px;'>"
                         + "#[[index]]. [[item.fact]]</div>")
                 .withProperty("fact", ValueProvider.identity()));
-        // end-source-example
 
         list.setId("chuck-norris-facts");
         addCard("Using templates", "List of random Chuck Norris facts", list);
     }
 
     private void createPeopleListWithDataProvider() {
         //@formatter:off
-        // begin-source-example
-        // source-example-heading: List of people with DataProvider
-        IronList<Person> list = new IronList<>();
+                        IronList<Person> list = new IronList<>();
```

Comment:
This line looks wrong, maybe remove `@formatter:off` above and then format again.